contracts/epochs/FlowEpoch.cdc

```diff
@@ -55,19 +68,18 @@ pub contract FlowEpochs {
         //
         // Node IDs are hex-encoded 32-byte arrays. Public keys are encoded as
         // by the flow-go crypto library, then hex-encoded.
-        nodeIDs: [String],
-        nodeRoles: [UInt8],
-        nodeStakingPubKeys: [String],
-        nodeNetworkPubKeys: [String],
-        nodeNetworkAddresses: [String],
+        nodeInfo: [FlowIDTableStaking.NodeInfo]
+
+        // The first view (inclusive) of the upcoming epoch.
+        firstView: UInt64,
 
         // The last view (inclusive) of the upcoming epoch.
         finalView: UInt64,
 
         // The cluster assignment for the upcoming epoch. Each element in the list
         // represents one cluster and contains all the node IDs assigned to that
         // cluster, separated by commas.
-        collectorClusters: [String],
+        collectorClusters: [FlowEpochClusterQC.Cluster],
```

Comment:
maybe adjust the comment here? This is no longer a string csv right?

Reply:
yes, it used to be a string, but we updated it. I'll fix the comment